### DISCUSSION TITLE
is possible create Server With user policy with no certificates?

### DISCUSSION THREAD

**Discussion by rafaone** [eclipse-milo/milo#1096] [General]
its not clear to me:
I created a simple server with:

[![enter image description here][1]][1]

and with user identity
[![enter image description here][2]][2]

but using the UaExpert I got [![enter image description here][3]][3]
 is possible to user the user token only for authenticate at begin ? or if we use user policy is mandatory the certificate?

  [1]: https://i.stack.imgur.com/sY6yw.png
  [2]: https://i.stack.imgur.com/dzWQ7.png
  [3]: https://i.stack.imgur.com/13W8a.png

**Comment by kevinherron** [comment]
If you allow user/password authentication then you should configure a certificate for the server, even if only unsecured endpoints will be used, because otherwise the password will be sent in plaintext.
